boa/code/version.py

```diff
@@ -0,0 +1,30 @@
+from boa.code.compiler import Neo2Compiler, Neo3Compiler
+from boa.code.converter import Neo2Converter, Neo3Converter
+
+
+class Version:
+    converter = None
+    compiler = None
+    __version = None
+
+    @staticmethod
+    def init(version):
+        if version is None:
+            raise Exception('Incompatible neo version: ' % version)
```

Comment:
This exception should be thrown if the version doesn't fullfill the ifs below aswell